### DISCUSSION TITLE
网站里面的备份列表的备份打开不能关闭

### DISCUSSION THREAD

**Discussion by yyg-max** [1Panel-dev/1Panel#6325] [Q&A]
### 联系方式

2144935044@qq.com

### 1Panel 版本

v1.10.14-lts

### 问题描述

网站里面的备份列表的备份打开不能关闭，关闭和x功能都没有作用

### 重现步骤

点击菜单中的网站，点击备份列表里面的备份打开就不能关闭。

### 期待的正确结果

能关闭

**Comment by wanghe-fit2cloud** [comment]
麻烦提供一下截图或者视频，在本地环境未能复现这个问题。
